src/layouts/AppContainer.tsx

```diff
@@ -147,7 +147,7 @@ class AppContainer extends React.Component<IProps, IState> {
   }
 
   private dismissNotif = (id: string) => () => this.props.dismissNotification(id);
-  private headerHtml = ( props: any ): any => <Header {...props} />;
+  private headerHtml = ( props: any ): any => <Header {...props} showRedemptionsButton={false} />;
```

Comment:
why the showRedemptionsButton here?

Reply:
fixed with:  https://github.com/daostack/alchemy/pull/1420/commits/6e7d4665d341999f477a1009d51ed9dbe48169c7